### PR TITLE
[Docs] Add Contribution Guidelines

### PR DESCRIPTION
**Pull request by macrigiuseppe**
This diff adds the following files:
- CONTRIBUTING
- DEVELOPER 
- CODE OF CONDUCT

### CODE REVIEW

File: DEVELOPERS.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+
+
+### Building Kepler.gl
+
+To build Kepler.gl, you clone the source code repository and use yarn to generate the non-minified
+and minified Kepler.gl files:
+
+```shell
+# Clone your Github repository:
+git clone https://github.com/<github username>/kepler.gl.git
```
Comment (heshan0131) [inline DEVELOPERS.md]: https protocol doesn't work for every one, use `git@` is safer

File: DEVELOPERS.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+# Go to the Kepler.gl directory:
+cd kepler.gl
+
+# Add the main Kepler.gl repository as an upstream remote to your repository:
+git remote add upstream "https://github.com/uber/kepler.gl.git"
```
Comment (heshan0131) [inline DEVELOPERS.md]: same here better with `git@`

File: DEVELOPERS.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+
+# Install JavaScript dependencies:
+yarn install
+
+# Build Kepler.gl:
+yarn build
```
Comment (heshan0131) [inline DEVELOPERS.md]: `npm start` here

File: DEVELOPERS.md [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+git clone https://github.com/<github username>/kepler.gl.git
+
+# Go to the Kepler.gl directory:
+cd kepler.gl
+
+# Add the main Kepler.gl repository as an upstream remote to your repository:
+git remote add upstream "https://github.com/uber/kepler.gl.git"
+
+# Install JavaScript dependencies:
+yarn install
```
Comment (heshan0131) [inline DEVELOPERS.md]: yarn --ignore-engines

File: DEVELOPERS.md [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+
```
Comment (heshan0131) [inline DEVELOPERS.md]: add the mapbox access token step

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# Contributor Code of Conduct
```
Comment (heshan0131) [inline CODE_OF_CONDUCT.md]: we have uber approved `CODE_OF_CONDUCT`

https://github.com/uber-go/zap/blob/master/CODE_OF_CONDUCT.md

**Comment by macrigiuseppe** [comment]
@heshan0131 thank you for reviewing this. I will update with your feedback.

**Pull request merged**